Modules/readline.c

```diff
@@ -1427,6 +1427,7 @@ PyInit_readline(void)
 
     PyModule_AddIntConstant(m, "_READLINE_VERSION", RL_READLINE_VERSION);
     PyModule_AddIntConstant(m, "_READLINE_RUNTIME_VERSION", rl_readline_version);
+    PyModule_AddStringConstant(m, "_rl_library_version", rl_library_version);
```

Comment:
Oh, I didn't notice that the variable on the previous line is in lower case but exported as UPPER case. I converted my new constant to UPPER case and rename "rl" to READLINE, as done for the 2nd constant.